### DISCUSSION TITLE
Easier way to merge all recruited units to the hero?

### DISCUSSION THREAD

**Discussion by HubKing** [ihhub/fheroes2#6792] [Q&A] [answered]
Let's say that your hero is in a castle with A,B,C,D,E types of troops. If you recruit A,B,C,D,E types of troops at once from well by pressing W --> M, other than manually clicking 10 times to merge them to the hero's troops, is there an easier way to do it at once?

**Comment by oleg-derevenetz** [comment, marked as answer]
IIRC the "down arrow" button should do it.
